lib/async_hooks.js

```diff
@@ -123,6 +124,35 @@ function createHook(fns) {
 }
 
 
+function getActiveResources() {
+  const handles = process._getActiveHandles();
+  const reqs = process._getActiveRequests();
```

Comment:
Could we... perhaps... move the implementations of these to an `internal/*` so we can more easily deprecate the `process._getActive*` variants later on?

Reply:
Yes, I'll do that once it is a little more settled if people are ok this idea